### PR TITLE
add images and about style

### PR DESCRIPTION
**Pull request by hectoruch**
Images and style from about page

### CODE REVIEW

File: _sass/components/_c-section-title.scss [hunk 1]
```diff
@@ -48,14 +42,8 @@
...
 
-  + p {
-    font-size: rem(16px);
-    line-height: rem(26px);
-    margin-top: rem(20px);
-  }
-
   @media screen and (min-width: $screen-s) {
-    padding-bottom: rem(50px);
+    padding-bottom: rem(0px);
```
Comment (agnlez) [inline _sass/components/_c-section-title.scss]: 0 is 0 in any unit. It's not necessary use the rem function here, just type 0 :)

File: _layouts/about.html [hunk 2]
```diff
@@ -30,9 +30,9 @@ <h1 class="c-section-title -light">{{ page.title }}</h1>
           {% for organization in organizations %}
             <div class="m-card--indexes">
               <a href="{{ organization.link }}" target="_blank" class="index-description">
-                <span style="background-image:url('{{site.baseurl}}/img/logos/{{ organization.logo }}')" alt=""></span>
+                <img src="{{site.baseurl}}/img/logos/{{ organization.logo }}" alt="{{ organization.name }} logo">
                 <h3 class="name">{{ organization.name }}</h3>
```
Comment (dhakelila) [inline _layouts/about.html:34]: Ops... I forgot to tell you. This is an span with a bg image for a reason.
As this images will be updated by the client, we can better control it size/proportion with a background. 
Can you back to the original version?

Sorry!

File: _sass/modules/_m-breadcrumbs.scss [hunk 3]
```diff
@@ -1,38 +1,27 @@
...
-    color: #5B5B5B;
-  }
-
+  position: relative;
+  text-align: center;
   a {
-    font-size: rem(9px);
-    font-family: $font-family-secondary;
+    color: #fff;
+    font-size: 10px;
```
Comment (dhakelila) [inline _sass/modules/_m-breadcrumbs.scss]: Use the vars, pelase!

File: _sass/modules/_m-breadcrumbs.scss [hunk 4]
```diff
@@ -1,38 +1,27 @@
...
-
   span {
-    padding: 0 rem(5px);
-  }
-
-  &.-light {
-    li, a {
-      color: $white;
-    }
+    color: #fff;
```
Comment (dhakelila) [inline _sass/modules/_m-breadcrumbs.scss]: Can you unify the common styles for <a> and <span>

File: _includes/breadcrumbs.html [hunk 5]
```diff
@@ -1,7 +1,5 @@
 <div class="m-breadcrumbs -light">
-  <ul>
-    <li><a href="{{ site.baseurl }}/">home </a></li>
-    <li><span> / </span></li>
-    <li> {{ page.page-name }} </li>
-  </ul>
+<a href="{{ site.baseurl }}/">home</a>
+<span> / </span>
+<span>{{ page.page-name }}</span>
```
Comment (dhakelila) [inline _includes/breadcrumbs.html]: Can you get the list back?
this is a list of links. 
: )

**Pull request merged**